i18n/zh/docusaurus-plugin-content-docs/current.json

```diff
@@ -4,43 +4,43 @@
     "description": "The label for version current"
   },
   "sidebar.documentationSideBar.category.Setup": {
-    "message": "Setup",
+    "message": "开始使用 KubeEdge",
     "description": "The label for category Setup in sidebar documentationSideBar"
   },
   "sidebar.documentationSideBar.category.Advanced Configuration": {
-    "message": "Advanced Configuration",
+    "message": "KubeEdge 高级配置",
     "description": "The label for category Advanced Configuration in sidebar documentationSideBar"
   },
   "sidebar.documentationSideBar.category.Architecture": {
-    "message": "Architecture",
+    "message": "KubeEdge 结构",
     "description": "The label for category Architecture in sidebar documentationSideBar"
   },
   "sidebar.documentationSideBar.category.Cloud Components": {
-    "message": "Cloud Components",
+    "message": "KubeEdge 云组件",
     "description": "The label for category Cloud Components in sidebar documentationSideBar"
   },
   "sidebar.documentationSideBar.category.Edge Components": {
-    "message": "Edge Components",
+    "message": "KubeEdge 边缘组件",
     "description": "The label for category Edge Components in sidebar documentationSideBar"
   },
   "sidebar.documentationSideBar.category.Developer Guide": {
-    "message": "Developer Guide",
+    "message": "KubeEdge 开发者向导",
     "description": "The label for category Developer Guide in sidebar documentationSideBar"
   },
   "sidebar.documentationSideBar.category.Mappers": {
-    "message": "Mappers",
+    "message": "KubeEdge 映射器",
```

Comment:
No need to translate the `Mapper`, thanks :)

Reply:
okay～